### PR TITLE
[RHDEVDOCS-5340] Bump versions to 1.11

### PR DESCRIPTION
**Pull request by krishnanshruthi**
**Do not merge before the Pipelines 1.11 GA, currently planned for June 15th!!!**

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/RHDEVDOCS-5340

Link to docs preview:
https://60832--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/using-pipelines-as-code.html#installing-pipelines-as-code-cli_using-pipelines-as-code

https://60832--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html

Download links don't work yet. They need to be checked shortly before merging. 

QE review: @ppitonak  , @rupalibehera 
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-installing-pipelines-as-code-cli.adoc [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
 
-You can install the `tkn pac` and `opc` version `1.10.0` binaries for the supported platforms:
+You can install the `tkn pac` and `opc` version `1.11.0` binaries for the supported platforms:
 
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.10.0/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.10.0/tkn-linux-s390x.tar.gz[Linux on {ibmzProductName} and {linuxoneProductName} (s390x)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.10.0/tkn-linux-ppc64le.tar.gz[Linux on {ibmpowerProductName} (ppc64le)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.10.0/tkn-macos-amd64.tar.gz[macOS]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.10.0/tkn-windows-amd64.zip[Windows]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.11.0/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
```
Comment (ppitonak) [inline modules/op-installing-pipelines-as-code-cli.adoc:14]: 1.10 binaries are uploaded on URL with both `pipeline` and `pipelines` in path. We could update it to `pipelines` in this doc for 1.11, it's closer to official product name.

**Review by ppitonak — CHANGES_REQUESTED** [review]

**Review by ppitonak — APPROVED** [review]

**Comment by shipsing** [comment]
@krishnanshruthi the changes look good to me.
But, I found 4 instances of 1.10 here: https://60832--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html#installing-tkn-on-linux-using-rpm

Do we need to update those instances to 1.11?
 subscription-manager repos --enable="pipelines-1.10-for-rhel-8-s390x-rpms" to  subscription-manager repos --enable="pipelines-1.11-for-rhel-8-s390x-rpms"

cc @ppitonak

**Comment by SNiemann15** [comment]
> @krishnanshruthi the changes look good to me. But, I found 4 instances of 1.10 here: https://60832--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html#installing-tkn-on-linux-using-rpm
> 
> Do we need to update those instances to 1.11? subscription-manager repos --enable="pipelines-1.10-for-rhel-8-s390x-rpms" to subscription-manager repos --enable="pipelines-1.11-for-rhel-8-s390x-rpms"
> 
> cc @ppitonak

Good catch @shipsing We missed those.

**Comment by ppitonak** [comment]
Yes, it needs to be updated, too.

**Comment by krishnanshruthi** [comment]
Replaced the 1.10 instances in https://60832--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html#installing-tkn-on-linux-using-rpm in the latest commit: https://github.com/openshift/openshift-docs/pull/60832/commits/b603cb4aac126d8e200011cc9d2a3b0d9fc71068

cc: @shipsing , @ppitonak , @SNiemann15

**Review by abrennan89 — COMMENTED** [review]
/lgtm

Confirming if QE have approved before merging

**Pull request merged**